boot.js

```diff
@@ -2373,6 +2373,54 @@ BrowserSystem.prototype.read = function read(location, charset, contentType) {
     return response.promise;
 };
 
+BrowserSystem.prototype.readAny = function readAny(locations, charset, contentType) {
```

Comment:
Recommend refactoring readAny in terms of read(), altering both the browser-system and system implementations of read() to normalize ENOENT and 404.